includes/CLI/Plugin_Check_Command.php

```diff
@@ -668,15 +673,22 @@ private function has_runtime_check( array $checks ) {
 	 * @param array $results          Check results.
 	 * @param int   $error_severity   Error severity level.
 	 * @param int   $warning_severity Warning severity level.
+	 * @param bool  $include_low_severity_errors Include less level of severity issues as warning.
```

Comment:
Can you fix the indentation so that these docs are aligned?

Reply:
yes!